recipes/pyomo.extras/meta.yaml

```diff
@@ -0,0 +1,46 @@
+# Do metapackages have a version?
+# Do metapackages have a license?
+
+package:
+  name: pyomo.extras
+  version: 3.0
+
+build:
+  noarch_python: True
+  number: 0
+
+requirements:
+  run:
+    - python
+    - ipython
+    - openpyxl
+    - sympy
+    - pymysql
+    - pyodbc
+    - pyro4
+    - pyyaml
+    - suds-jurko
+    - xlrd
+
+test:
+  imports:
+    - IPython
+    - openpyxl
+    - sympy
+    - pymysql
+    - pyodbc
+    - Pyro4
```

Comment:
The pyro4 package is imported at Pyro4 in Python.  This capitalization is
necessary.

On Fri, May 27, 2016 at 4:02 PM, jakirkham <notifications@github.com> wrote:

> In recipes/pyomo.extras/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/501#discussion_r64959614>
> :
>
> > +    - sympy
> > +    - pymysql
> > +    - pyodbc
> > +    - pyro4
> > +    - pyyaml
> > +    - suds-jurko
> > +    - xlrd
> > +
> > +test:
> > +  imports:
> > +    - IPython
> > +    - openpyxl
> > +    - sympy
> > +    - pymysql
> > +    - pyodbc
> > +    - Pyro4
>
> Also, please lower case.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/501/files/39842b5d5f0eae084256bfbb7ab07f154575f274#r64959614>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAsb-DiGGx4TTvu5spsAnAB2IWBuT_wFks5qF03IgaJpZM4IUwIm>
> .
>